### ISSUE TITLE
Interesting findings but some questions here

### ISSUE THREAD

**Issue by Turlan** [End2End-Diffusion/iREPA#1]
Thank you for the detailed experiments on what factors matter for representation alignment. However, I also noticed another paper from MiniMax titled “Towards Scalable Pre-training of Visual Tokenizers for Generation.” In their experiments, generation quality improves as linear probing performance increases, which appears to contradict your findings. How do you view this discrepancy?

**Comment by 1jsingh** [comment]
Hi @Turlan. Thanks for pointing this paper - very interesting.

I think the question here is causality vs correlation. VTP does encoder pretraining from scratch with combination of SSL (similar to dino), CLIP loss etc. As the model trains from scratch its expected that both linear probing and spatial structure improves -> which leads to better generation. Consider a dinov2 model trained from scratch. Its expected that as the model trains from 5k iter to convergence both LP and spatial structure continue to improve. However, this does not tell us exactly which one of global semantics or spatial structure is driving the gains. imo we need concrete ablations which is the focus of our analysis.
